easylist/easylist_adservers.txt

```diff
@@ -4198,6 +4220,8 @@
 ||gagacon.com^
 ||gagpetwet.com^
 ||gagwithholdoutsider.com^
+||gaietybeautify.com^
+||gaietybeautify.com^
```

Comment:
You have listed this one twice :smile:

Reply:
In any case, it's listed only once: https://github.com/easylist/easylist/blob/master/easylist/easylist_adservers.txt#L4227